fastlane/Fastfile

```diff
@@ -42,6 +42,9 @@ platform :ios do
       type: "adhoc",
       readonly: true
     )
+    increment_build_number(
```

Comment:
As part of the automation work done, there should be a point in which the automation is one as follow:
```
    udpated_build_number = increment_build_number(
      xcodeproj: project
    )
```
I see though that the type there is `adhoc` so perhaps this modifies another lane, not the release to appstore one 👍